### ISSUE TITLE
Add back in review stars on immersive article

### ISSUE THREAD

**Issue by HarryFischer** [guardian/dotcom-rendering#4700]
Stars were removed in [this PR](https://github.com/guardian/dotcom-rendering/pull/4671) . previous to this they were rendered on the page but hidden behind the headline box (not intentionally). Removed them for now to release that PR but would be good to add back inside the headline black box. Design to follow...

**Comment by abeddow91** [comment]
@HarryFischer stated we should wait on this one and do it alongside the immersive updates as it was part of the updates to H2s, pull quotes etc

**Comment by abeddow91** [comment]
closing in favour of https://github.com/guardian/dotcom-rendering/issues/11013
